com.coherentsolutions.training.auto.web.pashkovskaya/src/test/java/pageFacroty/PasswordPageFactory.java

```diff
@@ -0,0 +1,38 @@
+package pageFacroty;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.time.Duration;
+
+import static constans.LocatorsConstants.USERNAME_ON_HOME_PAGE;
+import static constans.WebElementsComstants.*;
+
+public class PasswordPageFactory {
```

Comment:
why factory? factory should produce object, in your case this class could have name 'PasswordPage'